### PR TITLE
Use Ui class as member instead of inheriting

### PR DESCRIPTION
**Pull request by asmaloney**

### CODE REVIEW

File: qCC/db_tree/ccPropertiesTreeDelegate.cpp [hunk 1]
```diff
@@ -60,6 +60,7 @@
 #include <QCheckBox>
 #include <QComboBox>
 #include <QHBoxLayout>
+#include <QLineEdit>
```
Comment (WargodHernandez) [inline qCC/db_tree/ccPropertiesTreeDelegate.cpp:63]: Why was this needed for the inheritance change?

Comment (asmaloney) [reply qCC/db_tree/ccPropertiesTreeDelegate.cpp:63]: Ah :-)

Because these UI header files include half of Qt (which is one of the reasons to move to the member instead of inheritance), QLineEdit was included indirectly in this file.

So when the UI header is moved out of the .h, there may be some fixes like this.
